install/prereqs/debian.rst

```diff
@@ -8,12 +8,20 @@ Debian or Ubuntu systems require the following packages:
 
 .. code-block:: bash
 
-   apt-get install bison ca-certificates \
+   apt-get install bison ca-certificates cmake flex gettext\
+           git libbz2-dev libfontconfig1 libglib2.0-dev \
+           libncurses5-dev libreadline6-dev \
+           libcurl4-openssl-dev libx11-dev libxrender1 \
+           libxt-dev m4 default-jre perl-modules zlib1g-dev \
+           curl git libbz2-dev make
+
+..   apt-get install bison ca-certificates \
            cmake flex g++ gettext git libbz2-dev \
            libfontconfig1 libglib2.0-dev libncurses5-dev \
            libreadline6-dev libssl-dev libx11-dev libxrender1 \
            libxt-dev m4 openjdk-8-jre \
            perl-modules zlib1g-dev \
+           curl git bzip2 make
```

Comment:
Again, I don't think we want this commented material.

Reply:
I agree